Makefile

```diff
@@ -24,6 +24,9 @@ all: docker
 gobuild:
 	./scripts/build false
 
+govet:
+	go vet $(shell go list ./agent/... | grep -v mocks)
```

Comment:
i tried to run this line one my mac, it shows `zsh: command not found: shell`, maybe you can remove the `shell`? shell should be used by default

Reply:
`shell` is a [make function](https://www.gnu.org/software/make/manual/html_node/Shell-Function.html).

Reply:
Can you try running `make govet` from your mac?   It works for me.

https://www.gnu.org/software/make/manual/html_node/Shell-Function.html